src/utils/l10n.ts

```diff
@@ -459,4 +459,63 @@ export const l10n = {
     displayMemoryUsageDescription:
       'Показать использование памяти на странице чата.',
   },
+  ca: {
+    attachmentButtonAccessibilityLabel: 'Enviar multimèdia',
+    emptyChatPlaceholder: 'Encara no hi ha missatges',
+    fileButtonAccessibilityLabel: 'Arxiu',
+    inputPlaceholder: 'Missatge',
+    sendButtonAccessibilityLabel: 'Enviar',
+    modelNotLoaded: 'Model no carregat. Si us plau, inicialitzeu el model.',
+    loadingModel: 'Carregant model...',
+    typeYourMessage: 'Escriviu el vostre missatge aquí',
+    conversationReset: 'Conversa reiniciada!',
+    networkError: 'Error de xarxa. Torneu-ho a intentar.',
+    goToModels: 'Anar a Models',
+    readyToChat: "Preparat per xatejar? Carregueu l'últim model utilitzat.",
+    pleaseLoadModel: 'Si us plau, carregueu un model per començar a xatejar.',
+    shortMemoryWarning: 'Advertència de memòria',
+    memoryWarning:
+      'Advertència: La mida del model està prop o supera la memòria total del dispositiu ({{totalMemory}} GB). Això pot causar comportaments inesperats com baix rendiment o inestabilitat del sistema.',
+    load: 'Carregar',
+    cancel: 'Cancel·lar',
+    download: 'Descarregar',
+    offload: 'Descarregar',
```

Comment:
I noticed both offload and download used the same term. I don't speak Catalan, so don't know if this is a mistake, or Descarregar would fit for both?

Reply:
It means the same in this context, just like in Spanish, as both words can be translated as 'descarregar'. However, we could use the word 'alliberar' for 'offload' to avoid confusion.

Reply:
Cool. Thanks.